data/human/marauders.txt

```diff
@@ -632,15 +632,15 @@ ship "Marauder Manta"
 		"A525 Atomic Steering"
 		"Hyperdrive"
 	
-	engine -33 38
-	engine 33 38
-	gun -66 -30 "Meteor Missile Launcher"
-	gun 66 -30 "Meteor Missile Launcher"
-	gun -27 -30 "Plasma Cannon"
-	gun 27 -30 "Plasma Cannon"
-	gun -20 -30 "Particle Cannon"
-	gun 20 -30 "Particle Cannon"
-	turret 0 -29 "Quad Blaster Turret"
+	engine -33 40
+	engine 33 40
+	gun -20 -31 "Particle Cannon"
+	gun 20 -31 "Particle Cannon"
+	gun -26.5 -30 "Plasma Cannon"
+	gun 26.5 -30 "Plasma Cannon"
+	gun -68.5 -33 "Meteor Missile Launcher"
+	gun 68.5 -33 "Meteor Missile Launcher"
```

Comment:
Any particular reason for moving the outermost (and now also the forwardmost) hardpoints to the end?

Reply:
I think because I copy pasted them from the ship file. It shouldn't actually change anything though.